kernel/packages/shared/catalogs/sagas.ts

```diff
@@ -171,8 +177,10 @@ function* fetchWearablesV2(filters: WearablesRequestFilters) {
         client
       )
       for (const { amount, definition } of ownedWearables) {
-        for (let i = 0; i < amount; i++) {
-          result.push(definition)
+        if (definition) {
```

Comment:
There are some corner cases where the definition might be null, so we are adding a check here